server/server.js

```diff
@@ -21,35 +22,50 @@ app.use(bodyParser.urlencoded({
 app.use('/static', express.static(path.join(__dirname, '/static')));
 
 
+
+var bobList = [];
+
+
+function nowPlusXMinutes(numMinutes) {
```

Comment:
This function is for setting an end time, as a length.
Ex: 
`startDate: Date.now()
endDate: nowPlusXMinutes()`.

Something like newDateNMinutesFromNow would be more explicit.
The better approach would be to use moment.js or some other time library.